server/paymentProviders/paypal/subscription.ts

```diff
@@ -280,7 +280,7 @@ export const setupPaypalSubscriptionForOrder = async (
   try {
     await paypalRequest(`billing/subscriptions/${paypalSubscriptionId}/activate`, null, hostCollective, 'POST');
     if (order.PaymentMethodId !== paymentMethod.id) {
-      order = await order.update({ PaymentMethodId: paymentMethod.id });
+      order = await order.update({ PaymentMethodId: paymentMethod.id, status: ORDER_STATUS.PROCESSING }); // Will be marked as active by the webhook
```

Comment:
Doing this will make sure the order is neither in the `ACTIVE` state (which would be false since it's not charged yet) nor in the `PAUSED` state (which is not the case either). Asked if that's safe in https://opencollective.slack.com/archives/C06K5J91675/p1717658997289129.